src/main/groovy/com/bmuschko/gradle/docker/tasks/image/Dockerfile.groovy

```diff
@@ -52,6 +52,26 @@ class Dockerfile extends DefaultTask {
         }
     }
 
+    void instructionsFromTemplate(File template) {
+        if (template.exists()) {
```

Comment:
Probably don't want to ignore if the file exists or not. I would check and throw a `FileNotFoundException`.